scripts/sublime-build/build.py.generic

```diff
@@ -1,5 +1,4 @@
-#!/Users/kbreit/Documents/Programming/ansible_collections/cisco/meraki/venv/bin/python
-
+#!/usr/bin/python
```

Comment:
If this needs to work using the venv's python nterpreter, it might be best to use `/usr/bin/env python`.